lib/releases.sh

```diff
@@ -1,44 +1,26 @@
 #!/bin/bash
 
 function get_latest_release() {
-  # get last page
+  # set url to get all releases from github
+  url="${1}?per_page=1000"
+  
+  # This gets the latest release as vx.y.z or vx.y.z-rc.0, including any version with a suffix starting with - , for example -rc.0
+  # The order is exactly as released in Github.
+  # Downside is that selecting official releases only isn't possible, while pre-release
+  # selection is possible given specific enough prefix, like v1.3.0-pre
+
   if [ -z "${GITHUB_TOKEN:-}" ]; then
-    last_page="$(curl -s -I "${1}" | grep '^link:' | sed -e 's/^link:.*page=//g' -e 's/>.*$//g')"
+    release_tag=$(curl -sL "${url}" | jq .[].name -r | egrep "^v1.4." -m 1)
```

Comment:
egrep is deprecated, use `grep -E` instead.

Reply:
done